prow/jenkins/jenkins.go

```diff
@@ -617,7 +622,7 @@ func (c *Client) BuildFromSpec(spec *prowapi.ProwJobSpec, buildID, prowJobID str
 
 // ListBuilds returns a list of all Jenkins builds for the
 // provided jobs (both scheduled and enqueued).
-func (c *Client) ListBuilds(jobs []string) (map[string]Build, error) {
+func (c *Client) ListBuilds(jobs []buildQueryParams) (map[string]Build, error) {
```

Comment:
If this is an exported method of the client we need to export the type so it's possible to use the client outside of this package